### PR TITLE
Add com.donhopkins.Micropolis

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] Please describe your application briefly.
> The X11 version of SimCity ported to Tcl/Tk by Don Hopkins with community patches
- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I have [built][build] and tested the submission locally.
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/SimHacker/micropolis/issues/90
- [X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://github.com/SimHacker/micropolis/pull/91

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build com.donhopkins.Micropolis

**Comment by Mailaender** [comment]
bot, build com.donhopkins.Micropolis

File: micropolis_git.patch
Comment (hfiguiere) [inline micropolis_git.patch:1]: this is really a huge patch. So big that github doesn't want to allow its review.

Comment (Mailaender) [reply micropolis_git.patch:1]: I can't inline it due to https://github.com/flatpak/flatpak-builder/issues/361.

**Comment by Mailaender** [comment]
bot, build com.donhopkins.Micropolis

File: micropolis_git.patch
Comment (Mailaender) [reply micropolis_git.patch:1]: I wanted it to be based on the original upstream, but I can base it onto the patch, true.

**Comment by Mailaender** [comment]
bot, build com.donhopkins.Micropolis

**Comment by bbhtt** [comment]
Can you also submit icon and metainfo upstream, please?

File: com.donhopkins.Micropolis.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+id: com.donhopkins.Micropolis
+runtime: org.freedesktop.Sdk
```
Comment (bbhtt) [inline com.donhopkins.Micropolis.yaml:2]: Why runtime is SDK?

Comment (Mailaender) [reply com.donhopkins.Micropolis.yaml]: Not sure where I copied this from.

**Comment by Mailaender** [comment]
> Can you also submit icon and metainfo upstream, please?

https://github.com/SimHacker/micropolis/pull/91

**Comment by Mailaender** [comment]
bot, build com.donhopkins.Micropolis

**Pull request closed without merging**